package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "wasm-imagemagick",
+  "version": "1.1.0",
+  "description": "Webassembly compilation of ImageMagick",
+  "main": "dist/magickApi.js",
+  "scripts": {
+    "test": "echo \"No tests :(\" && exit 1",
+    "postinstall": "npm i --no-save download-github-release && npx download-github-release knicknic wasm-imagemagick",
+    "prepare": "tsc --skipLibCheck --allowJs magickApi.js --outDir dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cancerberosgx/WASM-ImageMagick.git"
```

Comment:
should all url's be manually changed to https://github.com/KnicKnic/WASM-ImageMagick.git or will publishing take care of that?